code/modules/client/loadout/loadout_ooc.dm

```diff
@@ -20,3 +21,4 @@
 	C.inc_antag_token_count(1)
 	message_admins("[C.ckey] has purchased a genuine antag token.")
 	log_game("[C.ckey] has purchased a genuine antag token.")
+	..()
```

Comment:
this should be ran before the rest and the parent should return false/true just to not add tokens on failure (same with char slot)